android/src/main/java/com/urbanairship/reactnative/UrbanAirshipReactModule.java

```diff
@@ -74,11 +73,16 @@
     private static final String TAG_OPERATION_GROUP_NAME = "group";
     private static final String TAG_OPERATION_TYPE = "operationType";
     private static final String TAG_OPERATION_TAGS = "tags";
-
     private static final String TAG_OPERATION_ADD = "add";
     private static final String TAG_OPERATION_REMOVE = "remove";
     private static final String TAG_OPERATION_SET = "set";
 
+    private static final String ATTRIBUTE_OPERATION_KEY = "key";
+    private static final String ATTRIBUTE_OPERATION_VALUE = "value";
+    private static final String ATTRIBUTE_OPERATION_ACTION = "action";
```

Comment:
any chance we could call this ATTRIBUTE_OPERATION_TYPE/"type" so as not to further overload "action"?

Reply:
changed